samples/azure-graalvm-storage-samples-spring/pom.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.6.6</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-graalvm-storage-samples-spring</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>Azure Storage GraalVM Samples using Spring</name>
+  <description>Azure Storage Explorer sample</description>
+
+  <properties>
+    <java.version>11</java.version>
+    <spring-native.version>0.11.4</spring-native.version>
+    <graalvm.version>22.3.0</graalvm.version>
+    <native-maven-plugin.version>0.9.19</native-maven-plugin.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-sdk-bom</artifactId>
+        <version>1.2.10</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.37.0</version>
```

Comment:
I'm guessing this version is temporary and we are just waiting on the BOM to catch up?